src/js/components/TextInput/README 2.md

```diff
@@ -0,0 +1,389 @@
+## TextInput
```

Comment:
Not sure where this file came from, can you remove it? along with the rest of the files that end with `README 2.md`, any reason they were added?

Reply:
hmm not sure where thoses files came from, I noticed them after cleaning the cache and updating snapshots... deleted them now.